lib/kitchen/driver/oci/blockstorage.rb

```diff
@@ -20,7 +20,7 @@ module Kitchen
   module Driver
     class Oci
       # generic class for blockstorage
-      class Blockstorage < Oci
+      class Blockstorage < Oci # rubocop:disable Metrics/ClassLength
```

Comment:
Added this here because I noticed we have the same thing inside of `lib/kitchen/driver/oci.rb` so I figured it was okay but if not I can look at refactoring my methods again.